master/docs/developer/www.rst

```diff
@@ -617,6 +617,139 @@ Several methods are added to each "restangularized" objects, aside from get(), p
+
+Inject the ``bbData`` module to your application:
+
+  .. code-block:: javascript
+
+      angular.module('myApp', ['bbData'])
+
+Methods:
+
+* ``.get(endpoint, [id], [query])``: returns a promise<Collection>, when the promise is resolved, the Collection contains all the requested data
+
+  * call .getArray() on the returned promise to get the updating Collection before it's filled with the initial data
+
+  .. code-block:: coffeescript
+
+      # assign builds to $scope.builds once the Collection is filled
+      dataService.get('builds', builderid: 1).then (builds) ->
+          $scope.builds = builds
+          # load steps for every build
+          builds.forEach (b) -> b.loadSteps()
+
+      # assign builds to $scope.builds before the Collection is filled using the .getArray() function
+      $scope.builds = dataService.get('builds', builderid: 1).getArray()
+
+* ``.getXXX([id], [query])``: returns a promise<Collection>, when the promise is resolved, the Collection contains all the requested data
+
+  * it's higly advised to use these instead of the ``.get('string')`` function
+  * XXX can be the following: Builds, Builders, Buildrequests, Buildsets, Buildslaves, Changes, Changesources, Forceschedulers, Masters, Schedulers, Sourcestamps
+  * call .getArray() on the returned promise to get the updating Collection before it's filled with the initial data
+
+  .. code-block:: coffeescript
+
+      # assign builds to $scope.builds once the Collection is filled
+      dataService.getBuilds(builderid: 1).then (builds) ->
+          $scope.builds = builds
+          # load steps for every build
+          builds.forEach (b) -> b.loadSteps()
+
+      # assign builds to $scope.builds before the Collection is filled using the .getArray() function
+      $scope.builds = dataService.getBuilds(builderid: 1).getArray()
+
+* ``.open(scope)``: returns a DataAccessor, handles bindings, open a new accessor every time you need updating data in a controller
+
+  .. code-block:: coffeescript
+
+      # open a new accessor every time you need updating data in a controller
+      class DemoController extends Controller
+          constructor: ($scope, dataService) ->
+              # automatically closes all the bindings when the $scope is destroyed
+              opened = dataService.open($scope)
+              # alternative syntax:
+              #   opened = dataService.open()
+              #   opened.closeOnDestroy($scope)
+              # closing it manually is also possible:
+              #   opened.close()
+
+              # request new data, it updates automatically
+              @builders = opened.getBuilders(limit: 10, order: '-started_at').getArray()
+
+* ``.control(url, method, [params])``: returns a promise, sends a JSON RPC2 POST request to the server
+
+  .. code-block:: coffeescript
+
+      # open a new accessor every time you need updating data in a controller
+      dataService.control('Forceschedulers/force', 'force').then (response) ->
```

Comment:
this is 'forcescehedulers/force'.

This sounds a bit weird to have the control api use a REST style endpoint, while the get api is query like.